### DISCUSSION TITLE
about keyword "owning"

### DISCUSSION THREAD

**Discussion by lucianyao** [modular/modular#262] [Q&A]
Hi, I have seen a keyword "owning" in the example in [Types that support a “stealing move”](https://docs.modular.com/mojo/programming-manual.html#types-that-support-a-stealing-move)

I would like to know what the difference between it and the "owned"?  Is it just a typo?

Thanks!
Best regards
-Lucian

**Comment by lucianyao** [comment]
I guess it is a typo, I will post it as an issue report.

**Comment by lattner** [comment]
Yep it's a typo, will fix, thanks!
